activesupport/lib/active_support/test_case.rb

```diff
@@ -72,14 +72,5 @@ def test_order
     alias :assert_not_predicate :refute_predicate
     alias :assert_not_respond_to :refute_respond_to
     alias :assert_not_same :refute_same
-
-    # Fails if the block raises an exception.
-    #
-    #   assert_nothing_raised do
-    #     ...
-    #   end
-    def assert_nothing_raised(*args)
```

Comment:
Some history behind this method. http://blog.zenspider.com/blog/2012/01/assert_nothing_tested.html